ontology/GALLONT.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ontology_detail
+id: gallont
+title: "Plant Gall Ontology"
+contact:
+  email: adeans@psu.edu
+  github: adeans
+  label: Andy Deans
+  orcid: 0000-0002-2119-4663
+dependencies:
+  - id: pato
+  - id: po
+  - id: ro
+  - id: flopo
+  - id: caro
+  - id: obi
+  - id: poro
+  - id: ncbitaxon
+description: Ontology of plant gall phenotypes
+domain: phenotype
+homepage: https://github.com/adeans/gallont/tree/main
```

Comment:
Strange homepage link. Did you consider using the ODK documentation feature to generate a proper documentation page like https://pato-ontology.github.io/pato/?

Reply:
(not a requirement)

Reply:
> Strange homepage link. Did you consider using the ODK documentation feature to generate a proper documentation page like https://pato-ontology.github.io/pato/?

Oh yah. That would be preferable. I must've missed it in the ODK package. Is there documentation somewhere that would walk me through it? Somewhere in here? https://ontology-development-kit.readthedocs.io/en/latest/

Reply:
You add this to the ODK config:
https://github.com/obophenotype/human-phenotype-ontology/blob/53c4e0dcf3337e223c9afcd29dfb18e7ed372e7a/src/ontology/hp-odk.yaml#L182

then run update_repo twice

then set up your github pages like this: 
![image](https://github.com/OBOFoundry/OBOFoundry.github.io/assets/7070631/db88d85b-0668-44fb-94d4-37ccf39b15be)

Sorry for not having better docs on this :D

Reply:
Ok, I think I am close. Where do I "run update_repo twice"?

Reply:
https://oboacademy.github.io/obook/howto/odk-update/

Reply:
Thanks, @matentzn. I made progress (I think), but I get two QC errors now:

1. https://github.com/adeans/gallont/actions/runs/8266804229/job/22615690174 ("Error: Process completed with exit code 2.")
2. https://github.com/adeans/gallont/actions/runs/8266804231/job/22615690162 ("The "table-reader" plugin is not installed") 

Are these easy?

Reply:
The second seems to be a bug.

Lets start with the first https://github.com/adeans/gallont/actions/runs/8266804229/job/22615690174

This is a QC check that reflects the OBO foundries decision to fade out the dc elements namespace in favour of dc terms. So in your ontology, you need to replace all occurrences of the old dc elements namespace with dcterms.

The second can be circumvented by manually deleting this row:
https://github.com/adeans/gallont/blob/37daf2a543720392d16afb28b9031d0af6d8a796/mkdocs.yaml#L8

I would apprciate if you could make a bug report in the ODK repo though, as this needs to be fixed urgently.. :)